src/content/en/fundamentals/getting-started/primers/async-functions.md

```diff
@@ -0,0 +1,249 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: TODO
+
+{# wf_published_on: 2016-10-07 #}
+{# wf_updated_on: 2016-10-07 #}
+
+# Async functions - making promises friendly {: .page-title }
+
+{% include "web/_shared/contributors/jakearchibald.html" %}
+
+Async functions are enabled by default in Chrome 55 and they're quite frankly marvelous. They allow you to write promise-based code as if it were synchronous, but without blocking the main thread.
+
+Async functions look like this:
+
+    async function myFirstAsyncFucntion() {
+      try {
+        const resolvedValue = await aPromise;
+      }
+      catch (rejectedValue) {
+        // …
+      }
+    }
+
+If you use the `async` keyword before a function definition, you can then use `await` within the function. When you `await` a promise, the function is paused until the promise settles. If the promise fulfills, you get the value back. If the promise rejects, the rejected value is thrown.
```

Comment:
You could use this space to explicitly call out that top-level `await` [isn't supported](https://github.com/tc39/ecmascript-asyncawait/issues/9#issuecomment-127427447), and that the **only** time you can use `await` is inside an `async` function.

That's something that I found non-obvious.